examples/run_examples.py

```diff
@@ -62,18 +62,34 @@
 try:
     from daal4py.oneapi import sycl_context
     sycl_extention_available = True
-except:
+except ModuleNotFoundError:
     sycl_extention_available = False
+print('Sycl extensions available: {}'.format(sycl_extention_available))
 
 if sycl_extention_available:
     try:
         with sycl_context('gpu'):
             gpu_available = True
             availabe_devices.append("gpu")
-    except:
+    except RuntimeError:
         gpu_available = False
     availabe_devices.append("host")
     availabe_devices.append("cpu")
+    #validate that host and cpu devices avaialbe for logging reasons. Examples and
+    #vaidaton logic assumes that host and cpu devices are always available
+    try:
+        with sycl_context('cpu'):
+            cpu_available = True
```

Comment:
Why we cannot use `availabe_devices` list to detect host and device availability?

Reply:
for now avaialbe devices for cpu/host get assignment regardless actual support